staging/src/k8s.io/csi-api/pkg/apis/csi/v1alpha1/register.go

```diff
@@ -34,7 +34,8 @@ const (
 )
 
 // SchemeGroupVersion is group version used to register these objects
-var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
+var SchemeGroupVersionAlpha = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
+var SchemeGroupVersion = schema.GroupVersion{Group: GroupName, Version: "v1beta1"}
```

Comment:
v1beta1 resources do not belong in a v1alpha1 package. please create a new package and register the types via that package, following the normal registration pattern. that will also correctly generate clients and informers for the v1beta1 resources